django-stubs/contrib/admindocs/urls.pyi

```diff
@@ -1,3 +1,5 @@
-from typing import Any, List
+from typing import List, Union
 
-urlpatterns: List[Any] = ...
+from django.urls import URLPattern, URLResolver
+
+urlpatterns: List[Union[URLPattern, URLResolver]] = ...
```

Comment:
`Union[URLPattern, URLResolver]` might be a good candidate for a type alias. What do you think?

Reply:
Sure. Although the hard question is, where to put it and what to call it.

Right now I opted for `_AnyURL = Union[URLPattern, URLResolver]` in urls/__init__.pyi, but that doesn't seem too nice.

Reply:
`URLPatternOrResolver`?

Reply:
I actually like `_AnyURL` 🤔